### DISCUSSION TITLE
Final Assignment

### DISCUSSION THREAD

**Discussion by albertkun** [albertkun/211A-ASIAAM-191A#21] [Assignments]
### Due 8/1

Please use this thread to submit your [final group assignment](https://github.com/albertkun/211A-ASIAAM-191A/blob/master/final_assignment.md).

## Submission

- [Select a presentation slot on the final sheet](https://docs.google.com/spreadsheets/d/1svMIHd8KyPr7VmgU1bKC5wbRPS9sMtNnjXDgoTH5GMo/edit#gid=1026611838)
- Separately take the [individual group evaluation](https://forms.office.com/r/iifiQXNWUv)
- Post your group's repository and website link below:

ex.
Group Repo: https://github.com/akochaphum/my-awesome-group
Group website: https://akochaphum.github.io/my-awesome-group/index.html

**Comment by hrfrahn** [comment]
Group Repo: https://github.com/hrfrahn/191A-GroupProject
Group Website: https://hrfrahn.github.io/191A-GroupProject/

**Comment by luigimagdamit** [comment]
Group Repo: https://github.com/LuigiMagdamitUCLA/musical-lamp/tree/main
Group Website: https://luigimagdamitucla.github.io/musical-lamp/project/
Abita Venkatesh, Luigi Magdamit

**Comment by edmondywen** [comment]
Group Repo: https://github.com/edmondywen/asiaam-191a-la-heat-disparities
Group Website: https://edmondywen.github.io/asiaam-191a-la-heat-disparities/
Cynthia Orozco, Edmond Wen

**Comment by emilysuh** [comment]
Repo: https://github.com/emilysuh/alphabetmafia
Website: https://emilysuh.github.io/alphabetmafia/

**Comment by eandrusenki** [comment]
Repo: https://github.com/eandrusenki/wehearseebelieveyou
Website: https://eandrusenki.github.io/wehearseebelieveyou/titleixproject/
